atcoder/modint.hpp

```diff
@@ -206,6 +207,7 @@ template <int id> struct dynamic_modint : internal::modint_base {
 
     mint operator+() const { return *this; }
     mint operator-() const { return mint() - *this; }
+    operator bool() const { return _v != 0; }
```

Comment:
```suggestion
    explicit operator bool() const { return _v != 0; }
```
avoid implicit conversion